src/frontends/tensorflow/docs/architecture.md

```diff
@@ -0,0 +1,50 @@
+# OpenVINO TensorFlow Frontend Architecture and Workflow
+
+OpenVINO TensorFlow Frontend uses [TensorFlow Protobuf files](../src/proto) to read and to parse different TensorFlow model formats.
+The whole workflow can be split into two steps: model loading and conversion.
+
+```
+ov::frontend::tensorflow::FrontEnd FE;
```

Comment:
Can we add existed code like here: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/ir/docs/support_new_opset.md ?

Reply:
I still see hardcoded code :)

Reply:
Done.